### ISSUE TITLE
Add option to clear history for a given MUC

### ISSUE THREAD

**Issue by Fishbowler** [igniterealtime/openfire-monitoring-plugin#369]
Add option to Admin Panel to wipe the MAM history for a given room immediately.

Currently we only have a global option, and that also requires that the history is aged out, not immediately cleared.

Also for a given user??

**Comment by guusdk** [comment]
I don't think we should duplicate all admin console controls, by the way. If we add an option to clear room history in the 'core' admin console pages, then the Monitoring plugin's cleanup functionality should be triggered by that, too (if that plugin is loaded). Let's not add a 'clean' button in the core page, and another one in the Monitoring plugin page.
